webtau-junit5/src/test/java/com/twosigma/webtau/junit5/WebTauJunitExtensionTest.groovy

```diff
@@ -15,13 +18,47 @@ class WebTauJunitExtensionTest {
     @Test
     void "failed static init methods should have exception attached"() {
         runTest(WithFailedStaticInitAndClean)
-        def report = JavaReport.INSTANCE.create()
 
-        def tests = report.tests.stream().collect(Collectors.toList())
-        tests.should == ['*id'                    | 'failed' | 'errored'] {
-                         ________________________________________________
-                         'beforeAllfailingInit'   | false    | true
-                         'afterAllfailingCleanup' | false    | true  }
+        reportTests.should == ['*id'                    | 'failed' | 'errored'] {
+                               ________________________________________________
+                               'beforeAllfailingInit'   | false    | true
```

Comment:
nit: capitalize "Failing"

Reply:
That’s just the way we generate id. Not supposed to be human readable.